CMakeLists.txt

```diff
@@ -44,19 +42,13 @@ install(
   DESTINATION include
 )
 
-find_package(python_cmake_module REQUIRED)
-_ament_cmake_python_register_environment_hook()
-install(
-  DIRECTORY "src/${PROJECT_NAME}/"
-  DESTINATION "${PYTHON_INSTALL_DIR}/${PROJECT_NAME}"
-  PATTERN "*.pyc" EXCLUDE
-  PATTERN "__pycache__" EXCLUDE
-)
+ament_python_install_package(${PROJECT_NAME}
+  PACKAGE_DIR src/${PROJECT_NAME})
 
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
 ament_export_targets(${PROJECT_NAME})
-ament_export_dependencies(rclcpp rcutils builtin_interfaces)
+ament_export_dependencies(rclcpp rcutils)
```

Comment:
So this is a weird thing; what does `ament_export_dependencies` really mean?  Because technically this *package* depends on `builtin_interfaces`, but the C++ *library* does not.

Reply:
`ament_export_dependencies` is limited to propagating dependencies for consumption within CMake. So we could argue that while `builtin_interfaces` is a dependency of `message_filters`, it is not a dependency of the CMake part of `message_filters`.

I'm trying to think of a case where that breaks, but I can't come up with any realistic one.